Cargo.toml

```diff
@@ -37,10 +37,10 @@ atomic-polyfill = { version = "0.1.4" }
 atomic-polyfill = { version = "0.1.4" }
 
 [target.'cfg(target_arch = "avr")'.dependencies]
-atomic-polyfill = { version = "0.1.8", optional = true }
```

Comment:
Can we release a patch `heapless` with _just_ the `atomic-polyfill` update? This would allow us to release an updated `heapless` that still works with other heapless 0.7 releases.